### DISCUSSION TITLE
Deploying PocketBase on a Closed Network (Intranet) Using Windows Server

### DISCUSSION THREAD

**Discussion by matanshamshila** [pocketbase/pocketbase#3838] [Q&A]
I'm working for a company that has an intranet (closed network). I want to make the PocketBase database accessible through a public URL within the intranet, rather than just locally (http://127.0.0.1:8090/). I have a Windows computer that serves as a server for this purpose within the company. How can I accomplish this?

I've read that I might achieve it by setting up a reverse proxy, but I'm unsure if it's the best approach and exactly how to implement it. I'm familiar with web development but not with working with servers.

I would appreciate any advice on how to achieve this. Please also mention if there are any additional software required to be installed on the server to implement your solution

**Comment by Irvingouj** [comment]
this is a very generic question, is your firewall open? did you listen to all interface? try run ipconfig, and find a network interface that you wish to listen to and there's gonna be a ip address associated with it. other computer on the same local network can access it directly through http://your.ip.address:8090. Make sure your windows firewall has configured to accept incomming traffic at port 8090

**Comment by matanshamshila** [comment]
Thank you for trying to help
I've tried to do it by opening the firewall by it didn't worked for some reason
eventually I've been able to do it by setting up a reverse proxy using nginx

If anyone deals with it, I recommend to consider to use nginx
